### DISCUSSION TITLE
Spotlight

### DISCUSSION THREAD

**Discussion by fihnakis** [valinet/ExplorerPatcher#2411] [Q&A]
Not sure if it's the latest windows update or ExplorerPatcher but I no longer get the images updating. No matter the time frame I set for updating or even clicking the %SPOTLIGHTNEXT% to display a new image it's stuck on just one image. Anyone else having issues?

**Comment by pyrates999** [comment]
First uninstall EP, reboot, and see if this is still happening.

**Reply by fihnakis** [reply]
Thanks for the reply. I tried that. Also tried setting both background and lock screen to picture, rebooted, then back to spotlight. Tried dism and sfcscan, deregister and reregister spotlight. I tried a lot of things and none worked, lol. I finally gave up and loaded a reflect backup from January and spotlight is working again. Installed 23h2 update and was still working. Installed explorerpatcher and not working again.

**Reply by pyrates999** [reply]
ok, thank you for verifying this.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by fihnakis** [reply]
Win11 Pro version 23H2 (0S Build 22631.2506)

EP ver 22621.2428.59.1

Only settings I changed was taskbar centered with start button, always combine, show all taskbar icons, disable recommended section and that's it. 

I just did a full clean install of Win11 23H2 and the problem still persists. Also discovered if you try to add context menu item 'switch to next picture' or 'all items' none of the context menu options appear other than Learn about this picture.

Also just noticed if you choose the option disable win 11 context menu, all menu options when right clicking on 'Learn about this Picture' disappear other than Open.

**Reply by pyrates999** [reply]
can you show a screen shot of what happens when you right click when you disable the windows 11 context menu?

And please file this as a new issue as this is about spotlight.

**Reply by fihnakis** [reply]
I will do so next time I install EP. I was also having a lot of issues with the taskbar moving from center to the right near notification area and the start menu opening mostly off screen to the right when pressing windows key so I uninstalled EP.  Having some other win issues so need to eliminate those first in case they are causing all of this.

**Comment by pyrates999** [comment]
check this: https://www.windowslatest.com/2024/12/25/permanently-remove-learn-more-about-this-picture-icon-in-windows-11/
